packages/block-editor/src/hooks/block-bindings.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { store as blocksStore } from '@wordpress/blocks';
+import {
+	PanelBody,
+	__experimentalItemGroup as ItemGroup,
+	__experimentalItem as Item,
+} from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../lock-unlock';
+import InspectorControls from '../components/inspector-controls';
+import {
+	blockBindingsKey,
+	useBlockEditContext,
+} from '../components/block-edit/context';
+
+export const BlockBindingsPanel = () => {
+	const blockEditContext = useBlockEditContext();
+	const bindings = blockEditContext[ blockBindingsKey ];
```

Comment:
I believe we can actually get the bindings from the `metadata` attribute, which is passed to the props.

```
export const BlockBindingsPanel = ( { metadata } )
```

Reply:
Right, they are explicitly marked as exposed to the Edit panel through `attributeKeys`. That would further simplify and optimize code:

```js
export const BlockBindingsPanel = ( { metadata: bindings } ) => {
```